practice-one/src/index.html

```diff
@@ -42,5 +44,72 @@
         </nav>
     </header>
     <!-- End Header -->
+    <!-- Start footer -->
+    <footer class="footer-section">
+        <div class="footer-container">
+            <div class="footer-item-top">
+                <div class="item-top">
+                    <h4 class="text-primary text-white text-title-footer">contact us</h4>
+                    <address class="contact-us">
+                        <p class="text-primary description text-white opacity break">2300 W Arbutus Dr Citrus Springs, FL 34434, USA</p>
+                        <div>  
+                            <div class="text-primary description text-white opacity">+235 793-891-7938</div>
+                            <div class="text-primary description text-white opacity">contact@hofmannui.net</div>
```

Comment:
We should support clickable email

Reply:
I fixed it 
![image](https://github.com/HoangLiarPhuongLe/html_css_training/assets/110210274/38ffc9e6-b5ac-452d-8701-b8bd36e4b984)

Reply:
cool